iphone/Classes/TiApp.m

```diff
@@ -1175,12 +1175,14 @@ - (void)dealloc
 
 - (NSString*)systemUserAgent
 {
+#ifndef __clang_analyzer__
     UIDevice *currentDevice = [UIDevice currentDevice];
     NSString *currentLocaleIdentifier = [[NSLocale currentLocale] localeIdentifier];
     NSString *currentDeviceInfo = [NSString stringWithFormat:@"%@/%@; %@; %@;",[currentDevice model],[currentDevice systemVersion],[currentDevice systemName],currentLocaleIdentifier];
     NSString *kTitaniumUserAgentPrefix = [NSString stringWithFormat:@"%s%s%s %s%s","Appc","eler","ator","Tita","nium"];
-    
+    //ignore this on static analyzer
     return [[NSString stringWithFormat:@"%@/%s (%@)",kTitaniumUserAgentPrefix,TI_VERSION_STR,currentDeviceInfo] retain];
```

Comment:
ha. Sorry we didn't have to retain. I presumed you retained it on purpose in 
https://jira.appcelerator.org/browse/TIMOB-23623
but i just verified that without the retain, the sample app still builds correctly. changes made.